05-classification.Rmd

```diff
@@ -769,7 +791,7 @@ To better illustrate the problem, let's revisit the breast cancer data; except
 now we will remove many of the observations of malignant tumours, simulating
 what the data would look like if the cancer was rare. We will do this by
 picking only 3 observations randomly from the malignant group, and keeping all
-of the benign observations.
+of the benign observations. The new imbalanced data is shown in Figure \@ref(fig:05-unbalanced).
 
 ```{r 05-unbalanced, fig.height = 4, fig.width = 5, fig.cap = "Imbalanced data"}
 set.seed(3)
```

Comment:
This is another example of a seed we might want to make part of a more standard seed scheme, if possible.

Reply:
already an issue for this for a later global pass

Reply:
#168